recipe/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set version = "4.6.3" %}
+{% set version = "4.6.4" %}
 
 package:
   name: lxml
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/l/lxml/lxml-{{ version }}.tar.gz
-  sha256: 39b78571b3b30645ac77b95f7c69d1bffc4cf8c3b157c435a34da72e78c82468
+  sha256: daf9bd1fee31f1c7a5928b3e1059e09a8d683ea58fb3ffc773b6c88cb8d1399c
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
+    - python
+    - cython
```

Comment:
We do not need `python` and `cython` in the **build** section. They should be in the **host**